Research/ResearchUI/iOS/View Controllers/RSDWebViewController.swift

```diff
@@ -65,7 +65,7 @@ open class RSDWebViewController: UIViewController, WKNavigationDelegate {
     /// Convenience method for instantiating a web view controller that is the root view controller for a
     /// navigation controller.
     open class func instantiateController(using designSystem: RSDDesignSystem = RSDDesignSystem(), action: RSDWebViewUIAction? = nil) -> (RSDWebViewController, UINavigationController) {
-        let webVC = RSDWebViewController()
+        let webVC = self.init()
```

Comment:
thanks.

Note to self: This change allows using the class function to instantiate subclasses of RSDWebViewController.